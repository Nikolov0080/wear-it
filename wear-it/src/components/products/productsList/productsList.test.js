import React from 'react';
import renderer from 'react-test-renderer';
import ProductsList from './productsList';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Product List', () => {
    const tree = renderer
        .create(<BrowserRouter><ProductsList /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});