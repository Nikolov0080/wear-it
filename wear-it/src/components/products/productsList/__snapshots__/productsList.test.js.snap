// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Product List 1`] = `
<div
  className="box"
/>
`;
